libgnucash/engine/qofchoice.cpp

```diff
@@ -21,107 +21,96 @@
  *  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
  */
 
+#include <algorithm>
+#include <numeric>
+#include <vector>
+#include <string>
+#include <unordered_map>
 #include <config.h>
-#include <glib.h>
```

Comment:
This should be moved to qofchoice.h instead of removed altogether because several functions return `GList*`.